docs/packaging-applications/package-repositories/guides/container-registries/docker-hub.md

```diff
@@ -6,7 +6,7 @@ position: 50
 
 The default Docker Registry, which is maintained by the Docker organization, is the cloud-hosted [Docker Hub Registry](https://hub.docker.com/). This is the Registry which is used by docker engine when it is first installed and you call `docker search`.
 
-Searching for official public repositories do not require credentials. However searching for repositories of a non-official repository will require you to provide your Docker Hub username and password.
+As of September the 5th 2022 the Docker Hub Registry is [deprecating v1 endpoints](https://www.docker.com/blog/docker-hub-v1-api-deprecation/#:~:text=Docker%20has%20planned%20to%20deprecate,repositories%20on%20September%205th%2C%202022.) to retrieve tags and images. The equivalent v2 endpoints require authentication, therefore external feeds will require a username and password to access the Docker Hub API. Searching for repositories of a non-official repository will also require you to provide your Docker Hub username and password. Searching for official public repositories does not require credentials.
```

Comment:
> https://www.docker.com/blog/docker-hub-v1-api-deprecation/#:~:text=Docker%20has%20planned%20to%20deprecate,repositories%20on%20September%205th%2C%202022.
Maybe just a link to the page rather than the exact text? In case the text changes later. The quoted sentence is quite obvious on the page anyway.